dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -369,6 +369,49 @@ def diff(self, other_profile: NumericStatsMixin, options: Dict = None) -> Dict:
                 "and '{}'".format(cls.__name__, other_profile.__class__.__name__)
             )
 
+        len_self_bin_counts = 0
+        self_bin_counts = self._stored_histogram["histogram"]["bin_counts"]
+
+        try:
+            if self_bin_counts.all() is not None and len(self_bin_counts) > 0:
+                len_self_bin_counts = len(self_bin_counts)
+
+            # re-calculate `self` histogram
+            if not len_self_bin_counts == 10 and self_bin_counts.all() is not None:
+                histogram, hist_loss = self._regenerate_histogram(
+                    bin_counts=self._stored_histogram["histogram"]["bin_counts"],
+                    bin_edges=self._stored_histogram["histogram"]["bin_edges"],
+                    suggested_bin_count=9,
```

Comment:
`suggested_bin_count=9` but we check for `10`?

Reply:
changed back in most recent commit